src/core/db/manager.py

```diff
@@ -33,7 +35,13 @@ def __exit__(self, type, value, traceback):
 
 class DataHubManager:
 
-    def __init__(self, user, repo_base=None, is_app=False):
+    def __init__(self, user=settings.ANONYMOUS_ROLE, repo_base=None,
+                 is_app=False):
+
+        # blank users are set to anonymous role
+        if user == '':
```

Comment:
Oh ok. I guess that's part of the annoyance of passing around usernames instead of User objects. I'd say leave it like this then.